src/cljs/webchange/common/events.cljs

```diff
@@ -665,7 +669,8 @@
      
      (if block-user-interaction?
        (interactions/block-user-interaction)
-       (interactions/unblock-user-interaction))
+       (if (not= action-name "fx-dialog")
```

Comment:
Instead of hardcoding the name of the dialog please check if action has tag "fx". See  webchange.utils.scene-action-data action-tags